examples/tests/microbit_v2/main.c

```diff
@@ -0,0 +1,159 @@
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+#include <adc.h>
+#include <alarm.h>
+#include <ble.h>
+#include <button.h>
+#include <buzzer.h>
+#include <crc.h>
+#include <gap.h>
+#include <gpio.h>
+#include <led.h>
+#include <ninedof.h>
+#include <nrf51_serialization.h>
+#include <sound_pressure.h>
+#include <temperature.h>
+#include <timer.h>
+#include <tock.h>

```

Comment:
Double-check through here on which of these includes we actually use. For instance "nrf51_serialization.h" is definitely not used.

Reply:
You're right, I missed some includes when I was removing the unneeded ones.